libs/tof-lib/src/lib/fhirHelper.ts

```diff
@@ -235,10 +236,11 @@ export function codeableConceptHasCode(codeableConcept: ICodeableConcept, code:
 }
 
 export function setIgnoreWarningsValue(implementationGuide: IImplementationGuide, value: string) {
+  const Global = Globals;
```

Comment:
Is this needed? Can't the code below directly reference `Globals`?

Reply:
I checked on this multiple times and it just wasn't registering with the browser that Globals is available to be called within these functions even though it's imported in the fhirHelper script.  I think it has something to do with it being a library rather than something that lives server/client side.  It only started working when I set it locally.